### PR TITLE
Tests related to Fragment support

### PR DESCRIPTION
**Pull request by dmaurya929**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by dmaurya929** [comment]
The following scenarios are covered in tests for fragment support:
- Inserting fragment component in forms and sites. Test dialog behavior.
- Stitching of fragment in runtime and also reflection of model change.
- Repeatability with fragment
- Fragment container edit dialog behavior and hard check on edit dialog tabs.
- Embedding fragment
cc: @sakshi-arora1 @rismehta

File: ui.tests/test-module/specs/fragmentcontainer.spec.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        it ('check tabs and title in edit dialog', {retries: 3}, function() {
+            checkTitleInEditDialog(fragmentContainerEditPathSelector);
+            // stringent check on fragment container edit dialog tabs so that introducing new tab
+            // in form container edit dialog should force author to check if that is required for fragment
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab").should("have.length", 2);
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab-label:contains('Basic')")
+                .should("exist");
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab-label:contains('Data Model')")
+                .should("exist");
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab-label:contains('Submission')")
```
Comment (rismehta) [inline ui.tests/test-module/specs/fragmentcontainer.spec.js:119]: Can you try avoiding use of coral selector in the test, in fact rely on only our selectors ?

**Review by rismehta — CHANGES_REQUESTED** [review]
minor comments

**Comment by sakshi-arora1** [comment]
Merging @dmaurya929 pick the minor comment in next commit.

**Pull request merged**